core-bundle/contao/templates/_new/frontend_module/feed_reader.html.twig

```diff
@@ -0,0 +1,63 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
```

Comment:
We cannot extend from the content element base template as this would add the wrong CSS classes to the wrapper. In #5617 I added a new `_base.html.twig` for front end modules: https://github.com/contao/contao/pull/5617/files#diff-a6575ccf1263c730ea2b7a16a25de8c2b5c21ded553ccbef9ea24e8a1fed4bf6

May be we need a more general `@Contao/_base.html.twig`?

Reply:
+1 for the `frontend_module/_base.html.twig`

Reply:
I copied the base template from your pull request and use that one here, too.